modules/apps/frontend-js/frontend-js-tooltip-support-web/src/main/resources/META-INF/resources/index.tsx

```diff
@@ -170,7 +174,7 @@ const TooltipProvider = () => {
 			return delegate(document.body, eventName, SELECTOR_TRIGGER, () => {
 				dispatch({type: 'hide'});
 
-				restoreTitle(state.target);
+				restoreTitle(state.target as HTMLElement);
```

Comment:
I wasn't quite sure on how to type the reducer's initial state and added this here.

Reply:
Glad to see you were able to get rid of this cast.